es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const takeAChancePromise = takeAChance('Mike');
+
+takeAChancePromise
+  .then(value => {
+    console.log(value);
+  })
+  .catch(error => {
+    console.error(error);
+  });
```

Comment:
The `catch` should only print out the error message, not the whole stack dump:

---

![image](https://user-images.githubusercontent.com/16143771/161184912-0a5faac4-7973-4420-bfd4-687184e09ef2.png)

---

![image](https://user-images.githubusercontent.com/16143771/161184959-c88e8eeb-4046-4da4-a1f7-b28aad69e0cb.png)

Reply:
<img width="1501" alt="Screen Shot 2022-03-31 at 7 45 34 PM" src="https://user-images.githubusercontent.com/67080126/161185347-0b16c801-2e57-4191-af34-b36f5aa9b418.png">